Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodes.swift

```diff
@@ -19925,9 +19925,13 @@ public struct VersionTupleSyntax: SyntaxProtocol, SyntaxHashable {
   
   public init(
```

Comment:
Are you able to provide preconditions in this initializer to prevent bad input like `minorPeriod: nil, minor: nonNil`?

Reply:
I figured, but thought I'd ask just in case.

Reply:
I’m afraid a dutiful client will need to check that the period actually exists but in most cases I think it should be sufficient to just look at `major`, `minor` and `patch` and ignore the periods